### PR TITLE
Terror Spider tweaks

### PR DESCRIPTION
**Pull request by Kyep**
:cl: Kyep
fix: Empress of terror (admin-only mob) is no longer incorrectly prevented from laying brown spider eggs.
tweak: Admins are now able to spawn a queen of terror in a room without a vent. Previously, queens would not allow this. Now, this just results in an alert to admins (since its probably a mistake) and deactivates the mob AI (but leaves it available for player control).
add: If there are player controlled terror spiders, admins using check antags panel will now be able to see eggs/spiderlings currently growing on the station. These counters only count eggs/spiderlings on station, and ignore fake spiderlings.
fix: Empress of terror's 'erase brood' ability no longer incorrectly causes mothers of terror to spawn spiderlings on death.
sprite: Empress of terror now has its own sprite.
tweak: White spider infections have been nerfed a bit. They no longer provide passive healing or increase infection progress while their host/victim is in crit. Also, the first spiderling to emerge from an infection can no longer be a green. Finally, the eggs now self-delete after the third spiderling emerges, so even in the event they aren't treated with surgery, the host isn't left with confused movement forever.
/:cl:

Updated check antags panel:
check_antags_info

New sprite for empress of terror (purple one in the middle is a queen, for comparison):
terror_empress_new
Credit to FoS is already in the files.

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/terror_spiders/terror_spiders.dm [hunk 1]
```diff
@@ -214,6 +214,8 @@ var/global/list/ts_spiderling_list = list()
 				var/mob/living/carbon/human/H = G
 				if(!(H.dna.species.reagent_tag & PROCESS_ORG) || (!H.dna.species.tox_mod))
 					can_poison = 0
+				if(H.stat != DEAD && H.health < HEALTH_THRESHOLD_DEAD) // Intentionally not HEALTH_THRESHOLD_CRIT, as that is softcrit
```
Comment (Fox-McCloud) [inline code/modules/mob/living/simple_animal/hostile/terror_spiders/terror_spiders.dm]: may want to make this 

`if(!H.check_death_method() && H.health <= HEALTH_THRESHOLD_DEAD) H.death()`

Up to you, of course, but the above code basically will only call `death` on mobs that utilize the new crit method; mobs that use the old method would be immune to this.

I'd still make it `<=`, regardless of what you choose though.

Comment (Kyep) [reply code/modules/mob/living/simple_animal/hostile/terror_spiders/terror_spiders.dm]: Changed.

**Comment by TDSSS** [comment]
Could use some kind of flavour text/visible message when a spider kills someone that way, imo, describing just how the spider coupe de graces the victim.

**Comment by Kyep** [comment]
@TDSSS I thought about it, but I decided it wasn't really necessary, and probably wouldn't even be read given the volume of chat logs in combat.

In another note, this PR has been updated to give terror empresses a proper sprite (pictures in description).

**Comment by TDSSS** [comment]
The sprite looks kind of weird, the way it is hunched over. I assume it is to keep its size down but I'd expect it to be longer and less high/fat.

**Comment by Kyep** [comment]
There were references posted for the empress back in the day - its curved shape makes it look more evil, in my opinion.

**Comment by taukausanake** [comment]
Looks like an old hag-spider hunched up like that. It looks like there is some space to stretch it out. Bigger might be a bit more scary than how it is now. Having it curved doesn't look more evil, just off. And constipated...

**Comment by Kyep** [comment]
It pretty much IS an old hag-spider. Its an old empress.

**Comment by KasparoVy** [comment]
FoS killing it with the sprites again

**Comment by Kyep** [comment]
These sprites are from FoS, who was nice enough to make them for us.
We discussed it, and there just isn't any space to stretch out the sprite further.
Not to mention, I think it looks perfectly good as-is.
Due to size-induced mobs, (and the fact the bigger the sprite, the more work it takes) I am hesitant to make the sprite any bigger than this.

**Comment by Kyep** [comment]
Updates:

- Revert: This PR no longer makes terror spiders bypass newcrit.

- Tweak: White spider infections have been adjusted. They no longer provide passive healing or increase infection progress while their host/victim is in crit. Also, the first spiderling to emerge from an infection can no longer be a green. Finally, the eggs now self-delete after the third spiderling emerges, so even in the event they aren't treated, the host isn't left with permanently confused movement.

**Comment by MrMagolor** [comment]
This is just rambling, but I _do_ wish that terrors supplanted xenos entirely, as they are a more polished, balanced, and most importantly, unique (to Paradise) game-mode.

**Comment by TDSSS** [comment]
@MrMagolor 
I made a PR to supplant xenos with tspiders, but it was shot down with people saying we should just polish xenos to the same level instead. So, probably not happening

**Pull request merged**